_data/rules.yml

```diff
@@ -280,6 +280,11 @@ groups:
                 query: 'min_over_time(node_timex_sync_status[1m]) == 0 and node_timex_maxerror_seconds >= 16'
                 severity: warning
                 for: 2m
+              - name: Host requires reboot
+                description: '{{ $labels.instance }} requires a reboot.'
+                query: 'node_reboot_required > 0'
+                severity: warning
+                for: 2m
```

Comment:
I suggest increasing `for` to much longer period, like `4h` or more. Doing so would prevent over alerting when automatic remediation is in place (like apt auto reboot after install, or using things like [`kured`](https://github.com/weaveworks/kured) in kubernetes).

Additionally, in most cases, this is a cause-based informative alert and not a warning one, so `severity` should probably be lowered to `info`.

Reply:
@paulfantom I've made the associated changes.